scripts/cmake-build-mac.sh

```diff
@@ -23,12 +23,21 @@ _init
 
 ##### ### # Global definitions # ### ########################################
 ##### ### # Mac Qt # ### ####################################################
-MAC_QT_DIR=${QT_INSTALLATION_PATH}/${QT_VERSION_MAC}/clang_64
+if [[ -z "${MAC_QT_DIR}" ]]; then
+    MAC_QT_DIR=${QT_INSTALLATION_PATH}/${QT_VERSION_MAC}/clang_64
+else
+    MAC_QT_DIR="${MAC_QT_DIR}"
+fi
+
 MAC_QT_LIBRARYDIR=${MAC_QT_DIR}/lib
 
 ##### ### # Boost # ### #####################################################
 # Trying to find required Homebrew Boost libs
-BOOST_VERSION_MAC=1.73.0
+if [[ -z "${BOOST_VERSION_MAC}" ]]; then
+    BOOST_VERSION_MAC=1.73.0
+else
+    BOOST_VERSION_MAC="${BOOST_VERSION_MAC}"
```

Comment:
Same as above.

Reply:
Fixed in 89b74e4b138af412b82066b3e08a2d31f27a2ec5